File: Backend/EventManager/EventManager.Domain/Models/JobOfferStatus.cs
using EventManager.Domain.Enums;

namespace EventManager.Domain.Models
{
    public class JobOfferStatus
    {
        public int Id { get; set; }
        public Employee Employee { get; set; }
        public JobOffer JobOffer { get; set; }
        public OfferStatus Status { get; set; }
    }
}

File: Frontend/src/app/shared/models/job-offer-status.ts
import { OfferStatus } from './../enums/offer-status';
import { Employee } from './employee';
import { JobOffer } from './job-offer';

export class JobOfferStatus {
  id: number;
  employee: Employee;
  jobOffer: JobOffer;
  status: OfferStatus;

  constructor(id: number, employee: Employee, jobOffer: JobOffer, status: OfferStatus){
    this.id = id;
    this.employee = employee;
    this.jobOffer = jobOffer;
    this.status = status;
  }
}
